basictracer/span.py

```diff
@@ -44,14 +44,30 @@ def set_tag(self, key, value):
             self.tags[key] = value
         return super(BasicSpan, self).set_tag(key, value)
 
+    def log_kv(self, key_values, timestamp=None):
+        with self._lock:
+            self.logs.append(LogData(key_values, timestamp))
+        return super(BasicSpan, self).log_kv(key_values, timestamp)
+
     def log_event(self, event, payload=None):
+        key_values = {"event": event}
+        if payload is not None:
+            key_values["payload"] = payload
         with self._lock:
-            self.logs.append(LogData(event=event, payload=payload))
+            self.logs.append(LogData(key_values))
         return super(BasicSpan, self).log_event(event, payload)
 
     def log(self, **kwargs):
+        key_values = {}
```

Comment:
Reasonable, though the base class was doing the wrong thing :)

https://github.com/opentracing/opentracing-python/pull/37